src/test/java/com/exactpro/junit5/WithBufNameFormatter.java

```diff
@@ -0,0 +1,26 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+
+import java.util.Collection;
+
+class WithBufNameFormatter {
+
+    static String formatName(String methodName, Collection<ByteBuf> arguments) {
+        StringBuilder displayName = new StringBuilder(methodName);
+        for (ByteBuf buf : arguments) {
+            displayName.append(" [").append(formatBuffer(buf)).append("]");
+        }
+        return displayName.toString();
+    }
+
+    private static String formatBuffer(ByteBuf buf) {
```

Comment:
It's better to append to existing StringBuilder. Name can be changed to something like appendFormattedBuffer